libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -436,7 +443,7 @@ private object EvalExpression(string exp, ParserRuleContext context = null, stri
 
                 CheckExpressionResult(exp, error, result, templateName, context, errorPrefix);
             }
-            else if (result == null && !strictMode)
+            else if (result == null && lgOptions.StrictMode == false)
```

Comment:
`replaceNullKey` may has the higher priority than strictMode?

Reply:
yes. replaceNullKey  has the higher priority. If replaceNullKey applied, the result we obtained here will be a non null value, which will not throw exception under strictMode.